### PR TITLE
Add sanity check logs and prevent redirect if there isn't a valid token

### PR DESCRIPTION
**Pull request by davidpmccormick**
☝️

### CODE REVIEW

File: identity/webapp/pages/registration.tsx [hunk 1]
```diff
@@ -49,18 +49,20 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
 
     try {
       const token = decodeToken(sessionToken);
+      console.log({ token });
```
Comment (alexwlchan) [inline identity/webapp/pages/registration.tsx:52]: Aren't these tokens potentially sensitive? Do we want to be logging them?

File: identity/webapp/pages/registration.tsx [hunk 2]
```diff
@@ -49,18 +49,20 @@ export const getServerSideProps: GetServerSideProps<Props | AppErrorProps> =
...
-          destination: '/',
-          permanent: false,
-        },
-      };
+      // return {
+      //   redirect: {
+      //     destination: '/',
+      //     permanent: false,
+      //   },
+      // };
```
Comment (alexwlchan) [inline identity/webapp/pages/registration.tsx:65]: Why is this redirect commented out and not deleted?

Comment (melanierogan) [reply identity/webapp/pages/registration.tsx:65]: I was just catching up with David about this, I think we don't want to return a redirect in a catch, I reckon we'll want to throw and error and render that on the page in some way.

Comment (davidpmccormick) [reply identity/webapp/pages/registration.tsx:65]: we could redirect to `/error`?

File: identity/webapp/pages/registration.tsx (hunk 1 above)
Comment (alexwlchan) [reply identity/webapp/pages/registration.tsx:52]: Okay. If we do think it's okay to log them, let's log them with a bit more information – e.g.

```suggestion
      console.log(`Decoded post-registration token as $token`);
```

Comment (davidpmccormick) [reply identity/webapp/pages/registration.tsx:52]: Can do – but this change is only due to be ephemeral and using the shorthand object property name will make it clear that this is the `token` that's being logged

File: identity/webapp/pages/registration.tsx (hunk 2 above)
Comment (melanierogan) [reply identity/webapp/pages/registration.tsx:65]: Let's see if this is where the issue is. But I think we will ultimately need to catch and surface the error

**Review by melanierogan — APPROVED** [review]

**Pull request merged**